lib/active_merchant/billing/avs_result.rb

```diff
@@ -93,6 +98,22 @@ def to_hash
           'postal_match' => postal_match
         }
       end
+
+      def generate_avs_code(street_match:, postal_match:)
+        return if street_match.blank? && postal_match.blank?
```

Comment:
should this be `street_match.blank? || postal_match.blank?`

Reply:
No. If either `street_match` or `postal_match` is available, we can generate `A` or `P` codes respectively.

If `if street_match.blank? && postal_match.blank?` is true, these two letter codes offer no valuable information, and thus no `code` can be generated

Reply:
Although it can be said that `U` should get generated in those cases, but in various implementations, I found out that this made the most sense.